codegen/templates/generated.gotpl

```diff
@@ -226,6 +226,20 @@ func (ec *executionContext) FieldMiddleware(ctx context.Context, obj interface{}
 		switch d.Name {
 		{{- range $directive := .Directives }}
 		case "{{$directive.Name}}":
+			{{- if $directive.Implementation }}
```

Comment:
```
{{- with $directive.Implementation}}
    // args
    {{ . }}
{{-end}}
```
?

also the indenting went a bit funky here, aim for readable templates, gofmt will clean the code up.